rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java

```diff
@@ -94,6 +107,6 @@ private String removeSignature(String jws) {
 	 */
 	private boolean isValidCpcPlusOrg(Map<String, String> payloadMap) {
 		String orgName = payloadMap.get("name");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "orgName" which hides the field declared at line 25. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)

Reply:
Not keen on this check ^

Reply:
Same @clydet.

Reply:
Watch out @FlexionSonarQube, you've got a mutiny of your hands...  :laughing: